pkg/alertmanager/amcfg_test.go

```diff
@@ -78,15 +79,16 @@ func TestInitializeFromAlertmanagerConfig(t *testing.T) {
 
 	tests := []struct {
 		name            string
+		golden          string
```

Comment:
nit: may be move golden as last field, we follow a similar way in https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/prometheus/promcfg_test.go

Reply:
Thanks, I think I got to all the golden, pls review.